public/api/upload.php

```diff
@@ -15,10 +15,17 @@
 include 'cidFromSHA.php';
 
 $doenetId = mysqli_real_escape_string($conn, $_POST['doenetId']);
+$isActivityThumbnail = mysqli_real_escape_string($conn, $_POST['isActivityThumbnail']);
```

Comment:
Do we need both this API and the dedicated one for thumbnails?

Reply:
Taking a look at the APIs used, it looks like the two dedicated new ones are used, and there is an unchanged use of this one, but no one is taking advantage of this new feature in this API.